package.json

```
@@ -27,7 +27,7 @@
     "test-unit": "npm run build && jest test/unit/",
     "test-integration": "npm run build && jest test/integration",
     "test-travis": "jest --runInBand --testNamePattern='^((?!@slow).)*$' test/",
-    "test-unit-travis": "jest --runInBand test/unit/",
+    "test-unit-travis": "jest --runInBand test/unit/common.test.js",
```

Unintentionally disabling the other unit tests I think.
```suggestion
    "test-unit-travis": "jest --runInBand test/unit/",
```

---

[1c5baa7](https://github.com/IBM/cloudant-node-sdk/pull/721/commits/1c5baa7d5144eaab505ff1a1b29cafd888689340)